### ISSUE TITLE
experiments: add Hydra usage example

### ISSUE THREAD

**Issue by alex000kim** [iterative/example-repos-dev#167]
Need to showcase that DVC experiments support Hydra composition: https://dvc.org/doc/user-guide/experiment-management/hydra-composition

**Comment by daavoo** [comment]
Should we do this like in a new branch where we refactor to use Hydra or by default?

**Comment by alex000kim** [comment]
Not sure, tbh. Maybe a separate branch where we do a grid search over the learning rate and the batch size?

**Comment by daavoo** [comment]
> Not sure, tbh. Maybe a separate branch where we do a grid search over the learning rate and the batch size?

If it's only about grid-search, I already used it on the `tune-architecture` branch. Not sure how to showcase this as it is hidden in the `generate.sh`. I have tried adding it to the description in the P.R. like: https://github.com/iterative/example-get-started-experiments/pull/8

**Comment by alex000kim** [comment]
Maybe we should have a section e.g. "Things to try next" to the README and list hyperparam tuning with Hydra as one of the items?
